### PR TITLE
Allow to select the ci base image in PRs

### PR DESCRIPTION
**Pull request by mdellweg**
fixes #6834
https://pulp.plan.io/issues/6834

### CODE REVIEW

File: templates/travis/.travis/before_install.sh.j2 [hunk 1]
```diff
@@ -24,10 +27,12 @@ then
 {%- for repository in ["pulpcore", "pulp-smash", "pulp-openapi-generator"] + additional_plugins | map(attribute="name") | list %}
   export {{ repository | upper | replace("-", "_") }}_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/pulp\/{{ repository }}\/pull\/(\d+)' | awk -F'/' '{print $7}')
 {%- endfor %}
+  echo $COMMIT_MSG | sed -n -e 's/.*CI Base Image:\s*\([-_/[:alnum:]]*:[-_[:alnum:]]*\).*/ci_base: "\1"/p' >> .travis/vars/main.yaml
```
Comment (fao89) [inline templates/travis/.travis/before_install.sh.j2:30]: It would be cool to see it in the commit message of this PR

**Comment by daviddavis** [comment]
This is great. You may want to add a note about this to our docs after this section:

https://docs.pulpproject.org/contributing/git.html#requiring-other-pull-requests

File: templates/travis/.travis/before_install.sh.j2 (hunk 1 above)
Comment (mdellweg) [reply templates/travis/.travis/before_install.sh.j2:30]: i have an example here: https://github.com/pulp/pulp_container/pull/116

**Comment by mdellweg** [comment]
> This is great. You may want to add a note about this to our docs after this section:
> 
> https://docs.pulpproject.org/contributing/git.html#requiring-other-pull-requests
https://github.com/pulp/pulpcore/pull/781

File: templates/travis/.travis/before_install.sh.j2 (hunk 1 above)
Comment (fao89) [reply templates/travis/.travis/before_install.sh.j2:30]: cool!

**Pull request merged**